### DISCUSSION TITLE
Any benefits to using rotating BackConnect proxies vs. the traditional VPN/proxy setup I see people using?

### DISCUSSION THREAD

**Discussion by aellch** [jef/streetmerchant#2503] [Help]
I am working on getting my Streetmerchant setup, and to me it seems like using a BackConnect proxy is the way to go to avoid a ban + be able to have it check every half second to a second rather than the defaults people are using.  

For those unfamiliar, I'll give a cut and paste of the general description a provider I have dabbled with gives to describe the service: 

"Rotating proxies **automatically change IP on each HTTP request**, or every 3 or 15 minutes.
Ideal for tasks that require huge amounts of different IPs - scraping sites, traffic bots, bulk accounts registration, SEO tools, tickets and sneaker sites..."

Basically you set up one proxy on your device, and the proxy provider rotates IP's for each HTTP request.

They cost a bit more than your average proxy/VPN, but to me this seems like a better idea than using PIA or similar tools if you are trying to really get the best edge on everyone.

I'm just wondering if anyone has any experience using them with streetmerchant, and if they found value in doing so?  It seemms like the logical leg up to me, but I want to be sure I'm not missing something or overthinking it here as I am new to this type of thing. 

Any input would be greatly appreciated - as I am strongly leaning towards a VPS based setup with a BackConnect proxy configured. I think for now I'm just going to set it up on a local box with a VPN attached.... but I am really itching to use a BackConnect proxy instead.   

Thoughts and input?

**Comment by PlueSwe** [comment]
I would love to know more about this
